Packs/CloudflareZeroTrust/Integrations/CloudflareZeroTrustEventCollector/CloudflareZeroTrustEventCollector_description.md

```diff
@@ -0,0 +1,21 @@
+## Cloudflare Zero Trust
+
+_____
+Use this integration to fetch account audit logs, user audit logs, and access authentication logs from Cloudflare Zero Trust as events in Cortex XSIAM.
+_____
+
+#### **1. Retrieve Global API Key**
+
+- Visit [Retrieve Global API Key](https://developers.cloudflare.com/fundamentals/api/get-started/keys/).
```

Comment:
Why do I need to visit this page?
Maybe start with "To find how to retrieve a Cloudflare Global API Key, see [Retrieve Global API Key](https://developers.cloudflare.com/fundamentals/api/get-started/keys/).